extension/persistence/eclipselink/src/main/java/org/apache/polaris/extension/persistence/impl/eclipselink/PolarisEclipseLinkStore.java

```diff
@@ -62,14 +66,22 @@ public PolarisEclipseLinkStore(@NotNull PolarisDiagnostics diagnostics) {
     this.diagnosticServices = diagnostics;
   }
 
+  /** Initialize the store. This should be called before other methods. */
+  public void initialize(EntityManager session) {
+    PolarisSequenceManager.initialize(session);
+    initialized.set(true);
+  }
```

Comment:
Is this needed as one-time cleanup logic has been handled by `PolarisSequenceManager` already?
```
    if (!sequenceCleaned.get()) {
        sequenceCleaned.set(true);
   }
```

Reply:
Yes, it is needed. We need to initialize here so that if the POLARIS_SEQ check is going to fail, it does so outside the context of a transaction.

Reply:
Make sense, In that case, we should throw when initialization failed, so that we don't have to check whether it initialized.